src/pages/services/aem-assets-view/api/browse-view/index.md

```diff
@@ -0,0 +1,497 @@
+
+This method is called by the host application to determine which built-in actions are to be hidden.
+The host calls this method once whenever the resource selection changes.
+ 
+Extension code should ensure that this method returns fast because the host application blocks the rendering of the
+ActionBar until the actions' visibility could be determined.
+In particular it is recommended not to use backend server calls in this method.
+
+**Parameters:**
+- context (`string`): current [browsing context](#browsing-context)
+- **resourceSelection** (`object`): an object representing the resource selection
+  - resources (`array`): an array of selected resources.
+    - id (`string`): selected resource URN.
+    - path (`string`): selected resource path.
+
+**Returns** (`array`) an array of action ids that should be hidden from the ActionBar, or an empty array in case no action needs to be hidden
+
+**Example:**
+ ```js
+getHiddenBuiltInActions: ({ context, resourceSelection }) => {
+    return [];
+},
+```
+
+`overrideBuiltInAction({ actionId, context, resourceSelection })`
+
+**Description:**  Return true to indicate the Host should perform the built-in action, false otherwise. 
+
+This method is called by the Host when the user activates one of the built-in actions, before invoking the actual action
+handler. The method returns true if the Extension had performed custom action processing and the Host should not invoke
+built-in action handler. Otherwise the method call returns false, to indicate that the Extension
+had ignored the invocation and the Host should use built-in action handler.
+
+**Parameters:**
+- actionId (`string`): actionId built-in action id.
+- context (`string`): current [browsing context](#browsing-context).
+- **resourceSelection** (`object`): an object representing the resource selection.
+  - resources (`array`): an array of selected resources.
+    - id (`string`): selected resource URN.
+    - path (`string`): selected resource path.
+
+**Returns** (`boolean`) false for Host to use built-in action handler, true to skip built-in handler and stop
+
+**Example:**
+```js
+overrideBuiltInAction: ({ actionId, context, resourceSelection }) => {
+    //do some custom tasks
+    return true;  // skip the Host's built-in handler and stop  
+},
+```
+
+#### quickActions namespace
+
+The `quickActions` namespace include these 2 methods
+- `getHiddenBuiltInActions({ context, resource })`
+- `overrideBuiltInAction({ actionId, context, resource })`
+
+`getHiddenBuiltInActions({ context, resource })`
+
+**Description:**  returns an array of [built-in action id](#built-in-actions) that should be hidden in the specified context for the selected asset.
+
+This method is called by the host application to determine which built-in actions are to be hidden.
+The host calls this method once whenever an asset selection changes.
```

Comment:
Let's change "whenever an asset selection changes" to "whenever QuickAction menu is triggered"?